index.html

```diff
@@ -1271,12 +1271,18 @@ <h2>Verification Methods</h2>
 described in Section [[[#verification-methods]]]. When it is used in either
 place, its purpose is essentially the same; that is, it expresses one or more
 entities that are authorized to perform certain actions associated with the
-resource with which it is associated.
+resource with which it is associated. Note however that for [=verification methods=] it is only
+an assertion made by the `controller` of the [=controlled identifier document=] and this 
+assertion may be true or false. It is not necessarily a statement of fact. 
+To ensure that a [=verification method=] 
+is bound to a particular [=controller=], one must go from the expression of the [=verification method=]
+to its [=controlled-identifier-document=], and then ensure that the latter contains a reference to 
+the [=verification method=] and to its [=controller=]. 
             </p>
             <p>
 In the case of the [=controller=] of a [=controlled identifier document=], the
 [=controller=] can update the content of the document. In the case of the
```

Comment:
```suggestion
The [=controller=] of a [=controlled identifier document=]
can update the content of the document, while the
```

Reply:
I have accepted this change, but made it a complete sentence.